app/src/main/java/org/oppia/android/app/help/faq/FAQListActivity.kt

```diff
@@ -21,11 +21,6 @@ class FAQListActivity : InjectableAppCompatActivity(), RouteToFAQSingleListener
     faqListActivityPresenter.handleOnCreate()
   }
 
-  override fun onCreateOptionsMenu(menu: Menu?): Boolean {
```

Comment:
Just to check, why is this removed?

Reply:
The search_icon was part of `menu_faq_list_activity` which is an independent xml file. 

This function `onCreateOptionsMenu ` helps to bind the `menu_faq_list_activity` to the Activity/Screen so that we can show the `search` icon in header/toolbar.

As we have removed the `search_icon`, it means that `menu_faq_list_activity` will be empty, therefore we remove it too which means we don't need to bind it anymore to Activity.